test/test-v8debugapi.js

```diff
@@ -915,6 +916,30 @@ describe('v8debugapi', function() {
         });
     });
 
+    it('should capture without values when `allowExpressions=false`', function(done) {
+      // clone a clean breakpointInFoo
+      var bp = {id: breakpointInFoo.id, location: breakpointInFoo.location};
+      config.allowExpressions = false;
```

Comment:
That creates a new `config` object. This works by updating the values on the global config which is held by reference in the `api`. If we use `extend` we will need comparable lines replacing/restoring the config object held by the api.